### DISCUSSION TITLE
while opptimizing hyperparameter with optuna I get stuck in first trial and the trial 0 is not terminated

### DISCUSSION THREAD

**Discussion by Arch-suzuki-MB** [optuna/optuna#5459] [Q&A] [answered]
### Don't use GitHub Issues to ask support questions.

Hi, every one I try to optimize the hyperparameter of A2C and PPO from stable baseline 3 to applied for highway env using optuna. And I start for the A2C and I face an issue the progress bar is in 0% witch mean I still in trial 0 and the code is not moving for evaluating the policy. because I tried the same code in cartPole-v1 and actually is working fine it shown the results for each trials but for highway I don't know why it's instead stuck in the first trial and it's not end  for the highway env of gym

the output of the highway env 
![image](https://github.com/optuna/optuna/assets/99597954/87aa910c-f010-4dbf-91e6-b27ad87a99f9)
my code is in the following link:
https://colab.research.google.com/drive/1d9EwLjjgdlw2EAOv2sEVM4mFVdtOlp6j?usp=sharing

can any one @araffin , @nzw0301 , @hvy or any one from optuna team to help and thanks a lot

**Comment by nzw0301** [comment, marked as answer]
According to the logging message on colab when I ran your code, it takes time to perform some simulation called in `model.learn`; taking too much time to optimise the RL problem is unrelated to optuna. 

I'm not familiar with RL env, but it looks more complicated than cartPole-v1.
